shell/platform/darwin/macos/framework/Source/FlutterViewController.mm

```diff
@@ -410,6 +409,16 @@ - (void)dealloc {
 
 #pragma mark - Public methods
 
+- (FlutterView*)flutterView {
+  if (!self.viewLoaded) {
```

Comment:
Minor nit:
```suggestion
  if (!self.isViewLoaded) {
```

On iOS there's `-[UIViewController viewIfLoaded]` but there doesn't seem to be an AppKit equivalent.